frontend/src/components/activePage/TableTemplate/TableTemplateRow.tsx

```diff
@@ -78,20 +88,20 @@ const TableTemplateRow: FC<ITableTemplateRowProps> = ({ ...props }) => {
           size="middle"
           icon={<DeleteOutlined />}
           className="hidden lg:inline-block border-0"
-          onClick={e => {
-            e.stopPropagation();
-          }}
         >
-          Destory All
+          Destroy All
         </Button>
       </Popconfirm>
-
       <Dropdown
         trigger={['click']}
         overlay={
-          <Menu>
-            <Menu.Item icon={<DeleteOutlined className="text-lg" />} danger>
-              Destory All
+          <Menu onClick={click => dropdownHandler(click.key.toString())}>
+            <Menu.Item
+              key="Destroy All"
```

Comment:
Does not seem a great name for a key... 
I'd make an enum within this file, like DropDownAction, so you should be able to do 
```tsx
onClick={({key}) => dropDownHandler(key as DropDownAction)}
```